doc-content/enterprise-only/openshift/template-deploy-amq-proc.adoc

```diff
@@ -0,0 +1,27 @@
+[id='template-deploy-amq-{context}-proc']
+= Configuring communication with an AMQ server for {what_deploying_brief}
+
+If you use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver-amq.yaml` template file, the {KIE_SERVER} is configured to receive and respond to API messages using JMS, communicating with an AMQ server.
+
+If necessary for your environment, you can modify parameters for communication with the AMQ server.
+
+.Prerequisites
+
+* You started the configuration of the template, as described in {startlink}, using the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver-amq.yaml` template file.
+
+.Procedure
+
+Set any of the following parameters as required for your environment:
+
+* *AMQ Username* (`AMQ_USERNAME`) and *AMQ Password* (`AMQ_PASSWORD`): The user name and password of a standard broker user, if they are required to connect to the AMQ server.
+* *AMQ Role* (`AMQ_ROLE`): The user role for the standard broker user. The default role is `admin`.
+* *AMQ Queues* (`AMQ_QUEUES`): AMQ queue names, separated by commas. These queues are automatically created when the broker starts and are accessible as JNDI resources in the JBoss EAP server. If you use custom queue names, you must also set the same queue names in the `KIE_SERVER_JMS_QUEUE_RESPONSE`, `KIE_SERVER_JMS_QUEUE_REQUEST`, `KIE_SERVER_JMS_QUEUE_SIGNAL`, `KIE_SERVER_JMS_QUEUE_AUDIT`, and `KIE_SERVER_JMS_QUEUE_EXECUTOR` parameters.
+* *AMQ Global Max Size* (`AMQ_GLOBAL_MAX_SIZE`): The maximum amount of memory that message data can consume. If no value is specified, half of the memory available in the pod is allocated.
+* *AMQ Protocols* (`AMQ_PROTOCOL`): Broker protocols that the {KIE_SERVER} can use to communicate with the AMQ server, separated by commas. Allowed values are `openwire`, `amqp`, `stomp`, and `mqtt`. Only `openwire` is supported by JBoss EAP. The default value is `openwire`.
+* *AMQ Broker Image* (`AMQ_BROKER_IMAGESTREAM_NAME`): The image stream name for the AMQ broker.
+
+.Next steps
+
+If necessary, set additional parameters.
```

Comment:
what additional parameters?

Reply:
This is a compromise formula because it really means "other parameters in this section" but "this section" goes against certain style requirements. It's a long internal discussion we had.

Reply:
I understand but we can't really remove it right now as it's how the current structure works.